server/src/services/email/index.ts

```diff
@@ -0,0 +1,50 @@
+import nodemailer from "nodemailer";
+import emailVerificationTemplate, { VerifyEmailTemplateData } from './templates/email-verification-template'
+
+type EmailTemplate = "emailVerification";
+
+type EmailTemplateGenerators = { [k in EmailTemplate]: Function };
+
+type SendEmailData<T> = {
+  to: string,
+  data: T
+};
+
+const mailer = nodemailer.createTransport({
+  pool: true,
+  host: 'outlook.office365.com',
+  port: 587,
+  secure: false, // use TLS
+  auth: {
+    user: process.env.EMAIL_USER,
+    pass: process.env.EMAIL_PASSWORD
+  }
+});
+
+const templateGenerators: EmailTemplateGenerators = {
+  emailVerification: emailVerificationTemplate
+}
```

Comment:
```suggestion
const templateGenerators: EmailTemplateGenerators = {
  emailVerification: emailVerificationTemplate
};
```

Reply:
My prettier extension isn't working correctly with the project. 

I am able to get it working by making the following change .prettierrc `"parser": "babel"` => `"parser": "typescript"`. Is there a reason it is set to babel?